airflow/providers/amazon/aws/operators/emr.py

```diff
@@ -159,9 +160,11 @@ class EmrStartNotebookExecutionOperator(BaseOperator):
         group to associate with the master instance of the EMR cluster for this notebook execution.
     :param tags: Optional list of key value pair to associate with the notebook execution.
     :param waiter_countdown: Total amount of time the operator will wait for the notebook to stop.
-        Defaults to 25 * 60 seconds.
+        Defaults to 25 * 60 seconds. (Deprecated.  Please use waiter_max_attempts.)
     :param waiter_check_interval_seconds: Number of seconds between polling the state of the notebook.
-        Defaults to 60 seconds.
```

Comment:
Not strong opinion, but what if we move deprecated values to the bottom of docstring and `__init__` signature.
It should not affect anything because Operators only use keyword arguments however less important arguments would be in the bottom

Reply:
Sure, I suppose there's no harm in that.  I can make the change.